### DISCUSSION TITLE
Prevent "seeking" event when using setTime method

### DISCUSSION THREAD

**Discussion by jan-nick** [katspaugh/wavesurfer.js#3169] [Q&A] [answered]
The documentation says on the "seeking" event "When the user seeks to a new position". From this I expected the event to only fire when I interact with the element, but it triggers also when using the setTime method (and the seekTo method). Wouldn't it make sense not to trigger the event when using setTime?

**Comment by katspaugh** [comment, marked as answer]
The docs should be adjusted. It’s just a proxy for the media element’s seeking event and it will fire regardless of how you seek.

For user actions, there’s a click event.
